server/constants/transaction-kind.ts

```diff
@@ -1,6 +1,8 @@
 export enum TransactionKind {
   /** Transactions coming from the "Add Funds" */
   ADDED_FUNDS = 'ADDED_FUNDS',
+  /** Transactions from transferring the remaining balance from a project/event **/
```

Comment:
It's also for the balance of a collective to a fiscal host.

Reply:
Added `collective` to the comments as well. Thanks. 👍🏽